lgc/patch/ConfigBuilderBase.h

```diff
@@ -129,6 +130,9 @@ class ConfigBuilderBase {
   llvm::msgpack::MapDocNode m_graphicsRegistersNode;
   // MsgPack map node for graphics registers metadata
   // ".graphics_registers"
+  llvm::msgpack::MapDocNode m_computeRegistersNode;
+  // MsgPack map node for graphics registers metadata
+  // ".compute_registers"
```

Comment:
If comments are not on the same line, can you please put them *before* (and keep the line length in mind)? That is:
```suggestion
  // MsgPack map node for graphics registers metadata ".compute_registers"
  llvm::msgpack::MapDocNode m_computeRegistersNode;
```
I realize that the pattern already exists above -- I hadn't noticed it before, sorry.